BParkingNano/python/BToKLL_cff.py

```diff
@@ -99,8 +99,8 @@
         vtx_ez = ufloat('vtx_ez'),
         # Mll
         mll_raw = Var('userCand("dilepton").mass()', float),
-        mll_llfit = Var('userCand("dilepton").userFloat("fitted_mass")', float), # this might not work
-        mllErr_llfit = Var('userCand("dilepton").userFloat("fitted_massErr")', float), # this might not work
+#        mll_llfit = Var('userCand("dilepton").userFloat("fitted_mass")', float), # this might not work
```

Comment:
I would like keep these variables (mll_llfit and mllErr_llfit)

Reply:
changed